README.md

```diff
@@ -50,3 +50,68 @@ Useful command for local execution:
+
+## Configuration
+
+This section outlines the environment variables required for the application. Below are the variables, their descriptions, examples, and default values.
+
+### Required Environment Variables
+
+| Variable          | Description                               | Example                                                         |
+| ----------------- | ----------------------------------------- | --------------------------------------------------------------- |
+| `SPACES`          | Spaces to monitor. At least one required. | `kleros.eth:3000000,1inch.eth:6000000`                          |
+| `DB_URI`          | Postgres connection string                | `postgresql://user:password@localhost:5432/dbname`              |
+| `MAINNET_RPC_URL` | Provider URL for the Ethereum mainnet RPC | `https://mainnet.infura.io/v3/8238211010344719ad14a89db874158c` |
+
+- **SPACES format**: Spaces should be defined by the ENS and the starting block, with the following format: `kleros.eth:3000000`. Multiple spaces can be present, separated by commas.
+
+### Optional Environment Variables
+
+#### Debugging and Performance
+
+| Variable                | Description                      | Example                            | Default                            |
+| ----------------------- | -------------------------------- | ---------------------------------- | ---------------------------------- |
+| `DB_DEBUG`              | Print SQL queries to console     | `true`                             | `false`                            |
+| `MAX_BLOCKS_BATCH_SIZE` | Max blocks to process in a batch | `3000`                             | `200`                              |
+| `SNAPSHOT_GRAPHQL_URL`  | Snapshot GraphQL API endpoint    | `https://hub.snapshot.org/graphql` | `https://hub.snapshot.org/graphql` |
+
+#### Heartbeats
+
+Heartbeats are periodic GET requests performed against an endpoint to indicate that the service is alive.
+
+| Variable             | Description                               | Example                                         | Default |
+| -------------------- | ----------------------------------------- | ----------------------------------------------- | ------- |
+| `HEARTBEAT_URL`      | URL that will receive the GET request     | `https://instrumentation.keros.io/hooks/zodiac` |         |
+| `HEARTBEAT_INTERVAL` | Interval in milliseconds between requests | `1000`                                          | `60000` |
+
+### Transports Configuration
+
+For the bot being functional you should enable at least one transport, but all of them are optional and can be combined as desired. Transports are enabled when all the environment variables without default fallback are present with a valid value.
```

Comment:
What I understand is, you must have at least one, but can have more transports. If this is correct-

For me, this is clearer:
> For the bot being functional you should enable at least one transport. Transports are enabled when all the environment variables without default fallback are present with a valid value.

I.e:
> For the bot being functional you should enable at least one transport ~but all of them are optional and can be combined as desired~ Transports are enabled when all the environment variables without default fallback are present with a valid value.

At least one, but all are optional led me to have to re-read a couple of times

Reply:
@wadader That would be reasonable, but it doesn't have that check right now. Currently you can instantiate the bot with no transports (which would be useless). I should add that at least one transport is configured once I finish a few pressing features that we want to have for next week